### PR TITLE
states connected with manga control

### PR DESCRIPTION
**Pull request by fairnightzz**

### CODE REVIEW

File: src/components/MangaControl.tsx [hunk 1]
```diff
@@ -6,11 +6,30 @@ import { TitleIcon, BookmarkIcon, BookOpenIcon, CheckMarkIcon, ColorSwatchIcon,
...
+  chapters: string[],
+  chapter: string,
+  updateChapter: (newChapter: string) => void,
 }
 
 const MangaControl = (props: Props) => {
   const [menu, setMenu] = useState(true)
+  const { readerSettings, updateReaderSettings, chapter, chapters, updateChapter } = props;
+
+  const changeReaderMode = (event: React.ChangeEvent<HTMLSelectElement>) => {
```
Comment (MrPicklePinosaur) [inline src/components/MangaControl.tsx:19]: these event handlers are short enough, so you could probably just call them directly in the jsx as an anon function, doesn't matter too much though.

File: src/pages/Reader.tsx [hunk 2]
```diff
@@ -14,9 +14,10 @@ interface State {
...
   mangafile: Readonly<State>,
-  chapter: Readonly<State>,
+  chapter: string,
   page: Readonly<State>
-  readerSettings: ReaderSettings
+  readerSettings: ReaderSettings,
+  chapters: string[],
 }
 
 class Reader extends React.Component<any, State> { // fix typing up
```
Comment (MrPicklePinosaur) [inline src/pages/Reader.tsx:23]: why is this a class component again?

**Comment by MrPicklePinosaur** [comment]
LGTM

**Review by MrPicklePinosaur — APPROVED** [review]
LGTM

File: src/pages/Reader.tsx (hunk 2 above)
Comment (fairnightzz) [reply src/pages/Reader.tsx:23]: it was like that initially lol

**Pull request merged**